SQL/paradise_schema.sql

```diff
@@ -268,7 +268,7 @@ CREATE TABLE `player` (
   `toggles` int(11) DEFAULT NULL,
   `toggles_2` int(11) DEFAULT '0',
   `sound` mediumint(8) DEFAULT '31',
-  `volume` smallint(4) DEFAULT '100',
+  `volume_mixer` longtext COLLATE utf8mb4_unicode_ci DEFAULT NULL,
```

Comment:
`DEFAULT NULL` to avoid filling data redundantly. Column gets written on preferences update/volume adjustment.